test/expected/transformers/base-image-url.html

```diff
@@ -63,6 +63,14 @@
       <!--[if mso]></div></v:textbox></v:rect><![endif]-->
     </div>
 
+    <!--[if mso]>
+      <v:image src="https://example.com/image-2.jpg" xmlns:v="urn:schemas-microsoft-com:vml" style="600px;height:400px;" />
```

Comment:
Looks like the `width` property has been removed here (and on the following examples as well).

Reply:
Yup that was a mistake in the fixtures, updated now, thanks for spotting it!